src/Microsoft.HttpRepl.IntegrationTests/Commands/SetBaseCommandTests.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.HttpRepl.IntegrationTests.SampleApi;
+using Xunit;
+
+namespace Microsoft.HttpRepl.IntegrationTests.Commands
+{
+    public class SetBaseCommandTests : BaseIntegrationTest, IClassFixture<DualHttpCommandsFixture<SampleApiServerConfig>>
```

Comment:
Add a `WithSwaggerAndUnknownUri`?  I think we show a warning in that case.

Reply:
Looks like we write to console manager error instead of writing to console manager directly:
https://github.com/aspnet/HttpRepl/blob/507b52c18ddabdfa0491cad2cd4e664c4146f82c/src/Microsoft.HttpRepl/Commands/SetBaseCommand.cs#L41

So added unit test for the scenario:
https://github.com/aspnet/HttpRepl/commit/736dd109181feaa37eadbb9959d5cf30aa38dcb6